specifyweb/specify/migrations/0005_stratigraphy_age.py

```diff
@@ -0,0 +1,327 @@
+# Generated by Django 3.2.15 on 2024-08-02 06:04
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import specifyweb.specify.models
+from specifyweb.specify.update_schema_config import revert_table_schema_config, update_table_schema_config_with_defaults
+
+SCHEMA_CONFIG_TABLES = [
+    ('AbsoluteAge', None),
+    ('RelativeAge', None),
+    ('TectonicTreeDef', None),
+    ('TectonicTreeDefItem', None),
+    ('TectonicUnit', None),
+    ('RelativeAgeCitation', None),
+    ('RelativeAgeAttachment', None),
+    ('AbsoluteAgeCitation', None),
+    ('AbsoluteAgeAttachment', None),
+]
+
+def create_table_schema_config_with_defaults():
+    for discipline in specifyweb.specify.models.Discipline.objects.all():
+        for table, desc in SCHEMA_CONFIG_TABLES:
+            update_table_schema_config_with_defaults(table, discipline.id, discipline, desc)
+
+def revert_table_schema_config_with_defaults():
+    for table, _ in SCHEMA_CONFIG_TABLES:
+        revert_table_schema_config(table)
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('specify', '0004_schema_config_update'),
+    ]
+
+    def consolidated_python_django_migration_operations(apps, schema_editor):
+        create_table_schema_config_with_defaults()
+
+    def revert_cosolidated_python_django_migration_operations(apps, schema_editor):
+        revert_table_schema_config_with_defaults()
+
+    operations = [
+        migrations.CreateModel(
+            name='AbsoluteAge',
+            fields=[
+                ('id', models.AutoField(db_column='AbsoluteAgeID', primary_key=True, serialize=False)),
+                ('absoluteage', models.DecimalField(blank=True, db_column='AbsoluteAge', decimal_places=10, max_digits=22, null=True)),
+                ('agetype', models.CharField(blank=True, db_column='AgeType', max_length=64, null=True)),
```

Comment:
does this need to be handle like cotype and need for list creation in the migration?

Reply:
from control vocab (sedimentation, metamorphic, inclusion, original, fall,...)